apps/guardian-ui/src/components/setup/screens/verifyGuardians/VerifyGuardians.tsx

```diff
@@ -403,9 +403,7 @@ const WaitingForVerification: React.FC<{ verifiedConfigs: boolean }> = ({
 
   return (
     <Flex direction='row' height='100%' my={'auto'} gap={4} align='center'>
-      {verifiedConfigs ? (
-        <Text>{t('verify-guardians.all-guardians-verified')}</Text>
-      ) : (
+      {!verifiedConfigs && (
```

Comment:
Nit: This check is redundant, verifiedConfigs is already falsy

Reply:
just removed it and put just the spinner up out of that separate component